### ISSUE TITLE
Version update for ECW v3 files

### ISSUE THREAD

**Issue by PabloMeseguerEsbri** [DHI-GRAS/ecw2tiff#3]
Hi, amazing tool!

I just encountered this issue. Any suggestions to update the ECW SDK?

```
Converting file 1/2: xxx.ecw
ERROR 1: An error has occurred: Error 86 "File is invalid or corrupt"  file "" line 0
ERROR 1: Cannot open /input/xxxx.ecw which looks like a ECW format v3 file, that requires ECW SDK 5.0 or later
```

Thank you in advance. 

Pablo

**Comment by charalamm** [comment]
Hello @PabloMeseguerEsbri. Seems we will have to add the hexagon ECW 5.5 SDK. I am looking into this.

However it is not easy to come across ECW format v3 file samples. Could you share the file you are using or another file that gives the same output?

**Comment by PabloMeseguerEsbri** [comment]
Sorry, I can't share the file due to privacy issues. The problem is already solved as I used another tool. Thank for your assistance.
________________________________
De: Charalampos (cham) ***@***.***>
Enviado: martes, 30 de enero de 2024 9:17
Para: DHI-GRAS/ecw2tiff ***@***.***>
Cc: Pablo Meseguer ***@***.***>; Mention ***@***.***>
Asunto: Re: [DHI-GRAS/ecw2tiff] Version update for ECW v3 files (Issue #3)


Hello @PabloMeseguerEsbri<https://github.com/PabloMeseguerEsbri>. Seems we will have to add the hexagon ECW 5.5 SDK. I am looking into this.

However it is not easy to come across ECW format v3 file samples. Could you share the file you are using or another file that gives the same output?

**Comment by charalamm** [comment]
Fixed in dcf7a11fb7b2b56dad4cc7bdd0f571c1078599ae
